tests/e2e/specs/api/CloneGitRepoAPI.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { OpenshiftHelper } from '../../utils/OpenshiftHelper';
+import { expect } from 'chai';
+import { ShellString } from 'shelljs';
+import { GitUtil } from '../../utils/vsc/GitUtil';
+
+suite(`Empty Workspace export repository tests`, async function (): Promise<void> {
+    // works only for root user
+    const namespace: string = 'admin-devspaces';
+    const workspaceName: string = 'empty-' + Math.floor(Math.random() * 1000);
+    const gitRepository: string = 'https://github.com/crw-qe/web-nodejs-sample';
+    const clonedProjectName: string = GitUtil.getProjectNameFromGitUrl(gitRepository);
+    let containerWorkDir: string = '';
+
+    const openshiftHelper: OpenshiftHelper = new OpenshiftHelper(workspaceName, namespace);
+    const containerTerminal: OpenshiftHelper.ContainerTerminal = new OpenshiftHelper.ContainerTerminal(openshiftHelper);
+
+    const emptyYaml: string =
+        'apiVersion: workspace.devfile.io/v1alpha2\n' +
+        'kind: DevWorkspace\n' +
+        'metadata:\n' +
+        `  name: ${workspaceName}\n` +
+        'spec:\n' +
+        '  started: true\n' +
+        '  template:\n' +
+        '    components:\n' +
+        '      - container:\n' +
+        '          image: quay.io/devfile/universal-developer-image:latest\n' +
```

Comment:
It works for testing Eclipse Che only, not for Dev Spaces tests.

Reply:
Good catch

Reply:
field now parameterized with TS_DW_CONTAINER_IMAGE